### ISSUE TITLE
Potential new spatial standard

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#5181]
Was this already discussed? Discovered at TDWG - Along with IHO, Marine Regions has announced a new dataset to for marine areas: https://marineregions.org/files/newsletter_ecs.htm

_Originally posted by @Jegelewicz in https://github.com/ArctosDB/arctos/issues/5138#issuecomment-1283141637_

**Comment by Jegelewicz** [comment]
@sharpphyl thoughts?

**Comment by sharpphyl** [comment]
I think once we get through all the changes in [The Plan](https://github.com/ArctosDB/arctos/issues/5138), we should look at several of the [MarineRegions services](https://www.marineregions.org/downloads.php#marbound).  They have EEZs as well as the IHO seas - which I think Dusty has added.  The EEZs aren't as well defined as we might like, but could be helpful with offshore locations.  This new dataset of Extended Continental Shelves has potential as well.  Agree that this needs further discussion.

**Comment by genevieve-anderegg** [comment]
Agreed! This would be very useful for our collection as well as merits further discussion. With these services launching new datasets it would be a good idea to keep an eye on them. Perhaps some would make great "fun and flexible features" #5138
